docker-server.Dockerfile

```diff
@@ -43,10 +44,9 @@ RUN npm i -g ts-node  \
 USER nodejs
 
 COPY --from=pruner --chown=nodejs:nodejs ${WRKDR}/out/full/apps/${APP}/ .
+COPY --from=pruner --chown=nodejs:nodejs ${WRKDR}/out/full/packages/utils/ ./packages/utils
 COPY --from=builder --chown=nodejs:nodejs ${WRKDR}/node_modules/ ./node_modules
 
-ENV Env=prod
```

Comment:
Honor .env file. If we force prod, we will have to have prod MX credentials, which none of us have.